util/oidc/oidc_test.go

```diff
@@ -418,7 +418,8 @@ name: Test
 issuer: %s
 clientID: xxx
 clientSecret: yyy
-requestedScopes: ["oidc"]`, oidcTestServer.URL),
+requestedScopes: ["oidc"]
+skipAudienceCheckWhenTokenHasNoAudience: true`, oidcTestServer.URL),
```

Comment:
Default should be false. Where are you now testing with true?

```suggestion
skipAudienceCheckWhenTokenHasNoAudience: false`, oidcTestServer.URL),
```

Reply:
I needed this, because I’m now testing the return code of the `app.HandleCallback(w, req)` (which was previously ignored, masking the failure) and the OIDC mock server doesn’t set an audience on the token. Do you prefer that we modify the mock server to add an audience?

Reply:
This should be OK now.